### PR TITLE
Docs/readme

### LINKED ISSUE #57: Fill in readme

### PR DESCRIPTION
**Pull request by tylerarbus**
Close #57 
Check out the docs/readme branch to see the formatted version of the readme.

### CODE REVIEW

File: docs/TESTING.md [hunk 1]
```diff
@@ -2,25 +2,24 @@
...
-expect(foo).to.have.lengthOf(3);
-expect(tea).to.have.property('flavors').with.lengthOf(3);
+expect(foo).toBeInstanceOf('string');
+expect(foo).toBe('bar');
+expect(foo).toHaveLength(3);
+expect(tea).toHaveProperty('flavors');
 ```
 
-3. [Enzyme - React Component Testing](https://github.com/airbnb/enzyme)
+2. [Enzyme - React Component Testing](https://github.com/airbnb/enzyme)
```
Comment (terencetmac) [inline docs/TESTING.md]: Add Supertest

File: docs/TESTING.md [hunk 2]
```diff
@@ -2,25 +2,24 @@
...
 
 ## Running Tests
-`npm run test`
+`npm run test:dev`
```
Comment (terencetmac) [inline docs/TESTING.md]: Should we do 'running specific tests'

npm run test:db
npm run test:server
npm run test:client

File: package.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "",
-  "version": "",
+  "name": "Reflective",
+  "version": "1.0.0-beta",
   "description": "",
-  "author": "",
+  "author": "Jason Chang, Scott Clendening, Terence Mac, Tyler Arbus",
```
Comment (terencetmac) [inline package.json]: Terence Tham

File: package.json [hunk 4]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "",
-  "version": "",
+  "name": "Reflective",
+  "version": "1.0.0-beta",
   "description": "",
```
Comment (terencetmac) [inline package.json]: Virtual Journal that calls you once a day for 60s, converts your call to text entries and visualizes insights on the sentiment, emotions and keywords.

File: README.md [hunk 5]
```diff
@@ -1,53 +1,160 @@
...
+
+### Back-End
+
+1. Node
+2. Express
+3. PostgreSQL
+4. Kue
+5. JWT & Bcrypt
+6. Cron
+7. Elasticsearch
```
Comment (tylerarbus) [inline README.md]: see below

File: README.md [hunk 6]
```diff
@@ -1,53 +1,167 @@
...
-- etc
+- Elasticsearch 5.3.x
+
+You may use a local or hosted database/search service for Redis, Postgresql, and Elasticsearch.
+
+### APIs
+
+Reflective uses third-party services to power its calling and analytics features. To use the application, you'll need to generate API keys from the following services:
+
+- [Twillio](https://www.twilio.com/)
```
Comment (smclendening) [inline README.md]: Twilio only has one 'l'

File: README.md [hunk 7]
```diff
@@ -1,53 +1,167 @@
...
 ```sh
-npm install -g bower
 npm install
-bower install
 ```
 
-### Roadmap
+### Testing
+
+Our test suite is build with **Jest, Enzyme** (for testing React components), and **Supertest** (for mock HTTP requests). To run all tests:
```
Comment (smclendening) [inline README.md]: Should 'build' be 'built'?

**Pull request merged**